pandas/core/tools/datetimes.py

```diff
@@ -244,15 +256,15 @@ def _return_parsed_timezone_results(result, timezones, tz, name):
 
 def _convert_listlike_datetimes(
     arg,
-    format,
-    name=None,
-    tz=None,
-    unit=None,
-    errors=None,
-    infer_datetime_format=None,
-    dayfirst=None,
-    yearfirst=None,
-    exact=None,
+    format: Optional[str],
+    name: Label = None,
+    tz: Optional[str] = None,
```

Comment:
tz could be a tzinfo?  might merit a type in pd._typing corresponding to "tzinfo or str that we can convert to one"